catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -112,11 +112,14 @@
 
     private static final int DEFAULT_MAX_START_INDEX = 50000;
 
+    private static final int DEFAULT_THREAD_POOL_SIZE = 128;
+
     private static Logger logger = LoggerFactory.getLogger(CachingFederationStrategy.class);
 
     private final SolrCache cache;
 
```

Comment:
This is not going to work if we use blueprint injection like we had planned to do. We should instead create the ExecutorService in an `init()` method after all the dependencies have been injected.